src/yb/yql/pggate/pggate_flags.cc

```diff
@@ -35,6 +35,9 @@ DEFINE_UNKNOWN_int32(pggate_ybclient_reactor_threads, 2,
 DEFINE_UNKNOWN_string(pggate_master_addresses, "",
               "Addresses of the master servers to which the PostgreSQL proxy server connects.");
 
+DEFINE_UNKNOWN_string(pggate_cert_base_name, "",
+              "Certificate base name computed by pg_wrapper");
+
```

Comment:
```suggestion
DEFINE_NON_RUNTIME_string(pggate_cert_base_name, "",
              "Certificate base name computed by pg_wrapper");

```
New flags need to be labeled RUNTIME or NON_RUNTIME, UNKNOWN ones are just uncategorized flags introduced before we started labeling them.